chemprop/cli/common.py

```diff
@@ -139,6 +145,21 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
 
 
 def process_common_args(args: Namespace) -> Namespace:
+    for key in ["atom_features_path", "atom_descriptors_path", "bond_features_path"]:
+        inds_paths = getattr(args, key)
+        ind_path_dict = {}
+        if inds_paths:
```

Comment:
Style suggestion: switch these two lines so that `ind_path_dict` is only instantiated if it will be used.

Reply:
Better yet, I change this to this to decrease nesting

```
def process_common_args(args: Namespace) -> Namespace:
    for key in ["atom_features_path", "atom_descriptors_path", "bond_features_path"]:
        inds_paths = getattr(args, key)
        
        if not inds_paths:
            continue

        ind_path_dict = {}
        
        for ind_path in inds_paths:
            if len(ind_path) == 1:
                ind = 0
                path = ind_path[0]
            else:
                ind, path = ind_path

            ind_path_dict[int(ind)] = Path(path)

        setattr(args, key, ind_path_dict)

    return args
```